### PR TITLE
Introduce AngularJS to Oh-Hell.

### PR DESCRIPTION
**Pull request by martinsuchen35**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -17,8 +17,10 @@
     "node"
   ],
   "private": true,
-  "devDependencies": {
+  "dependencies": {
```
Comment (victor-xiong) [inline bower.json]: I would still prefer to use devDependencies, as bootstrap is depending on jquery, so we kind of get jquery for free.

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "url": "https://github.com/victor-xiong/oh-hell/issues"
   },
   "homepage": "https://github.com/victor-xiong/oh-hell#readme",
-  "devDependencies": {
+  "dependencies": {
```
Comment (victor-xiong) [inline package.json]: same as the last one

File: .bowerrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "directory": "app/bower_components"
```
Comment (victor-xiong) [inline .bowerrc]: agree on using 4 spaces as our coding standard?

Comment (martinsuchen35) [inline .bowerrc]: Agreed, nice catch.

**Pull request merged**